lib/src/builder/parsing/members_from_ast.dart

```diff
@@ -168,6 +168,18 @@ class _BoilerplateMemberDetector {
       return;
     }
 
+    final rightHandSide = node.variables.firstInitializer;
+    if (rightHandSide is MethodInvocation && rightHandSide.methodName.name == 'uiFunction') {
```

Comment:
Also, I think we should look for nested calls to this, to support wrapping these in HOCs inline. For example:

```dart
UiFactory<FooProps> Foo = connect(
 ...
)(uiFunction({
});
```

We might want to swap this out for 
```dart
anyDescendantIdentifiers(rightHandSide, (identifier) => identifier.name == 'uiFunction');
```

Can we also add a test case for that?